pkg/metadata/host/host_tags.go

```diff
@@ -93,7 +93,7 @@ func GetHostTags(ctx context.Context, cached bool) *Tags {
 	}
 
 	hname, _ := hostname.Get(ctx)
-	clusterName := clustername.GetClusterName(ctx, hname)
+	clusterName := clustername.GetRFC1123CompliantClusterName(ctx, hname)
```

Comment:
This should use `GetClusterNameTagValue` as it generates the `kube_cluster_name` tag.

Reply:
indeed, good catch

Reply:
```suggestion
	clusterName := clustername.GetClusterNameTagValue(ctx, hname)
```